ivy/functional/frontends/paddle/nn/functional/activation.py

```diff
@@ -195,14 +195,9 @@ def relu_(x, name=None):
 
 @with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
 @to_ivy_arrays_and_back
-def elu(
-    x,
-    /,
-    *,
-    alpha=1.0,
-    name=None,
-):
-    return ivy.elu(x, alpha=alpha)
```

Comment:
Why did you remove the existing implementation? I think the ivy.elu (https://github.com/unifyai/ivy/blob/f5f8cd107865b7de6650e495474d0667e8ec4757/ivy/functional/ivy/experimental/activations.py#L408) is working alright?